static/js/jquery.min.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.12.4 | (c) jQuery Foundation | jquery.org/license */
```

Comment:
This is a super old version

Reply:
Yes, but considering we should allow some old Explorers to download，I have to use it——Just like we have to use “var” in the "main.js".

Reply:
@nschonni: Sorry, I remember we'd discussed about the version of browsers for ages...but now, it seems we've only suppoorted IE >=10... https://github.com/nodejs/nodejs.org/pull/2602. It's been fixed.